src/client/datascience/jupyter/kernels/cellExecution.ts

```diff
@@ -677,9 +677,11 @@ export class CellExecution {
                 }
                 // tslint:disable-next-line:restrict-plus-operands
                 existing.data['text/plain'] = formatStreamText(concatMultilineString(`${existingOutput}${newContent}`));
+                traceInfoIf(!!process.env.VSC_JUPYTER_LOG_KERNEL_OUTPUT, `Append Output '${existing.data['text/plain']}'`);
                 edit.replaceCellOutput(this.cell.index, [...exitingCellOutput]); // This is necessary to get VS code to update (for now)
             } else {
                 const originalText = formatStreamText(concatMultilineString(msg.content.text));
+                traceInfoIf(!!process.env.VSC_JUPYTER_LOG_KERNEL_OUTPUT, `Insert Output '${originalText}'`);
```

Comment:
This helped figure out why the tests were failing.
Leaving for now, as I have another test thats flaky (stderr & stdout test).
Also this stuff never gets logged in production & this variable is only initialized in a specific test, hence no mark for now.